services/app-web/src/pages/CMSDashboard/RateReviewsDashboard/RateReviewsTable.tsx

```diff
@@ -134,6 +144,42 @@ const getSelectedFiltersFromUrl = (
     return filterValues as FilterOptionType[]
 }
 
+const getDateRangeFilterFromUrl = (
+    columnFilters: ColumnFiltersState,
+    id: string
+): string | undefined => {
+    const filterLookup: { [key: string]: string[] } = {}
+    columnFilters.forEach(
+        (filter) => (filterLookup[filter.id] = filter.value as string[])
+    )
+
+    if (filterLookup[id]) {
+        return filterLookup[id][0]
+    }
+    return undefined
+}
+
+// Even though both date filters are similar we don't want to combine into one function because we don't always know
```

Comment:
I feel like these should be one function because they are filtering the same columnn and we pass in its id